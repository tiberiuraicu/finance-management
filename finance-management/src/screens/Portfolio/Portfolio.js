import React, { useState, useEffect } from "react";
import { Text, View } from "react-native";
import CustomButton from "../../components/CustomButton";
import WrapperScrollView from "../../components/WrapperScrollView";
import styles from "./Portfolio.style";

const Portfolio = ({ navigation }) => {
  return (
    <WrapperScrollView>
      <Text style={styles.portofolioValueText}>Portfolio value</Text>
      <CustomButton
        onPress={() => {
          navigation.navigate("AddHoldings");
        }}
        text="ADD HOLDINGS"
      ></CustomButton>
      <View style={styles.holdingsList}></View>
    </WrapperScrollView>
  );
};

export default Portfolio;
