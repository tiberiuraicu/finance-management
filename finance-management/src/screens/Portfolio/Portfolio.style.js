import { StyleSheet } from "react-native";

export default StyleSheet.create({
  holdingsList: {
    height: "100%",
    marginTop: "5%",
    paddingTop: "5%",
    paddingBottom: "10%",
    borderTopStartRadius: 25,
    borderTopEndRadius: 25,
    backgroundColor: "#F2FCFE",
  },
  portofolioValueText: {
    fontSize: 22,
    fontWeight: "bold",
    alignSelf: "center",
    color: "white",
    marginTop: "5%",

  },
});
