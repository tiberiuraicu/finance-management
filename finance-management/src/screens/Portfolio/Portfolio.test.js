import Portfolio from "./Portfolio.js";
import { shallow } from 'enzyme';

describe('Protfolio screen tests', () => {
    it('renders the component', () => {
      const wrapper = shallow(<Portfolio />);
      expect(wrapper).toMatchSnapshot();
    });
  });