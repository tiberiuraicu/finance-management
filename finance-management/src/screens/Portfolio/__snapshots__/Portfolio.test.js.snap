// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Protfolio screen tests renders the component 1`] = `ShallowWrapper {}`;
